themes/default/layouts/shortcodes/cloud-intro.html

```diff
@@ -1,16 +1,26 @@
 <div>
     <p>
         Pulumi's infrastructure-as-code SDK helps you create, deploy, and manage {{ .Get 0 }} containers,
-        serverless functions, and infrastructure using real programming languages. The Pulumi {{ .Get 0 }} provider packages
+        serverless functions, and infrastructure using <a href="{{ relref . "/docs/intro/languages" }}">imperative  programming languages</a>. The Pulumi {{ .Get 0 }} provider packages
         and <a href="{{ relref . "/docs/reference/cli" }}">CLI</a>
         help you accomplish all these within minutes.
     </p>
     <p>
-        This guide will have you quickly provisioning infrastructure on {{ .Get 0 }} with Pulumi in your
-        favorite language.
-    </p>
-    <p>
-        To get started we will walk through installing Pulumi, installing your preferred language runtime (if needed),
-        and configuring Pulumi with {{ .Get 0 }}. Click the button below to get started.
+        For a quick example of how Pulumi deploys infrastructure on {{ .Get 0 }}, this tutorial takes you through the following steps to easily deploy a static website:
     </p>
+    <ol>
+        <li>Setting up and configuring Pulumi to access your {{ .Get 0 }} account.</li>
```

Comment:
To get around some of the duplication, you could capture the cloud into a variable:

```
{{ $cloud := .Get 0 }}

<div>
    <p>
        ....

        {{ if eq $cloud AWS }}
            ...
```

Reply:
Good idea. Incorporated this in https://github.com/pulumi/pulumi-hugo/pull/241/commits/ae5047f42a3b41791a220e11b98138b310c110ef